script.js

```diff
@@ -167,3 +167,19 @@ form.addEventListener('submit', (event) => {
     form.submit();
   }
 });
+
+/* Local Storage */
+
+const formStorage = document.querySelector('#form');
+formStorage.addEventListener('submit', () => {
```

Comment:
- [ ] Nice work using a single object to store all the data. :+1: Currently data is getting saved when the form is submitted. But the project requirements suggest that we should save the data when the text inside the inputs change. Kindly update it to match the project requirements. ✍️

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/225636564-a2a0c345-6954-4fa0-99b9-75fa4660870d.png)